src/fights.ts

```diff
@@ -665,8 +677,11 @@ const pygmySniffed = () =>
   sniffSources.some((source) => pygmyBanishHandlers.some(({ pygmy }) => pygmy === get(source)));
 
 const pygmyMacro = Macro.step(
-  ...pygmyBanishHandlers.map(({ pygmy, skill, item }) =>
-    Macro.if_(pygmy, skill ? Macro.trySkill(skill).item(item) : Macro.item(item)),
+  ...pygmyBanishHandlers.map(({ pygmy, skill, item, check, limit }) =>
+    Macro.if_(
+      pygmy,
+      skill && get?.(check) < limit ? Macro.trySkill(skill).item(item) : Macro.item(item),
```

Comment:
```suggestion
      skill && get?.(check) < limit ? Macro.trySkill(skill).item(item) : Macro.item(item),
```
I think I don't quite understand what's happening here. `get` is a function that's always defined, right? why are we using optional chaning on it?

Reply:
In general, this seems like you're trying to do the 3-arg version of `Macro.externalIf`; I recognize that it doesn't behave super nicely with possibly-nullish parameters, but I think that would still be better